services/jenkins-autoscaling/lambda_mxnet_ci/autoscaling/handler.py

```diff
@@ -417,7 +417,7 @@ def _unconnected_instances(nodes: list, instance_uptime: Dict[str, int], ec2_res
     dict_starting_nodes: Dict[str, List[str]] = defaultdict(list)
     instances_filter = ec2_resource.instances.filter(
         Filters=[
-            {'Name': 'instance-state-name', 'Values': ['pending', 'running']},
+            {'Name': 'instance-state-name', 'Values': ['starting', 'running']},
```

Comment:
Why remove pending?

Reply:
https://code.amazon.com/packages/Mxnet-ci/blobs/ed4fdb464790b6ccb0a0aa3f3d8f6b64b452a6f8/--/autoscaling/lambda_mxnet_ci/autoscaling/handler.py#L422
I tried to keep it same as this
But I'm guessing this should be `pending` too

Reply:
You also said that there is no state called starting?

Reply:
Ya. Switching it back to pending.